### PR TITLE
added content to the file

### PR DESCRIPTION
**Pull request by YFL-dev**
-  Created command.md. 
- Added content most used 11 shell command to the file.

### CODE REVIEW

File: command.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+`cp` copy a file to destination directory
+`cp -r` copy a directory and everything in it to destination directory
+
+### 7. mv
+`mv` mv + [file name 1] + [file name 2] is used to rename file name 
+     mv + [file name] + [directory name] is used to move file
+
+### 8. rm
+`rm` remove files
+`rm -r` recursive remove files
```
Comment (92fueler) [inline command.md]: `rm -rf` is used more often than `rm -r`, but use it with cautious

**Review by 92fueler — COMMENTED** [review]
lgtm 
but the next would be check out how to install packages on ubuntu.

File: command.md (hunk 1 above)
Comment (YFL-dev) [reply command.md]: +1

File: command.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+`cp` copy a file to destination directory
+`cp -r` copy a directory and everything in it to destination directory
+
+### 7. mv
+`mv` mv + [file name 1] + [file name 2] is used to rename file name 
+     mv + [file name] + [directory name] is used to move file
+
+### 8. rm
+`rm` remove files
+`rm -rf` recursive remove files
```
Comment (92fueler) [inline command.md]: `rm -r` and `rm -rf` are different. You can still keep `rm -r`

File: command.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+`sudo + [command]` execute the command as root user
+
+### 10. cat
+`cat` print input content to output
+`cat file1 file2 > file` combine file1 and file 2 to one file
+
+### 11. clear
+`clear` clear screen
+
+### 12. apt
```
Comment (92fueler) [inline command.md]: usually it has `sudo` before apt.

**Comment by 92fueler** [comment]
fix those two and you're good to go

**Comment by 92fueler** [comment]
Commit messages are usually meaningful. Plz don't use 1st, 2nd, etc as your commit messages

File: command.md (hunk 2 above)
Comment (YFL-dev) [reply command.md]: +1

File: command.md (hunk 3 above)
Comment (YFL-dev) [reply command.md]: +1

**Review by 92fueler — APPROVED** [review]
lgtm

**Pull request merged**